### DISCUSSION TITLE
Adding New Users

### DISCUSSION THREAD

**Discussion by dwhacks** [magicbug/Cloudlog#1771] [General]
I've finally updated my install... not sure for how long it wasn't updating, but my VM was on an older Ubuntu LTS so I also upgraded that and started fresh. Got my user all set up and everything seems to be working as it should, with many improvements! (The whole thing seems faster...)

Anyway, I see that now, as of 2.0, cloudlog supports multi users/stations/logbooks. Am I understanding this correctly? Someone other then me can have their own personal logbook in my cloudlog instance?

Assuming this is the case, is the only way to add more users is for an Admin to create the user/password and then give it to the other person to set up and hopefully they change the password? Or is there some sort of "sign up" page for new users. 

I'm assuming there is not a signup page, as this may create a bit of a mess if some sort of bot just created endless users...

**Comment by magicbug** [comment]
Currently the admin will need to create the account, its on the roadmap to make this public via an option but haven't coded it yet

**Reply by dwhacks** [reply]
Thanks, this is excellent!
